### PR TITLE
Add macOS URL support for more browsers

### PR DESCRIPTION
**Pull request by wrgoto**
Adds URL support for other Chrome-based browsers on macOS:

- Brave Browser Beta
- Sidekick
- Opera

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you add beta/canary versions of the existing browsers there too? While you're at it.

**Comment by sindresorhus** [comment]
You also need to update the docs (readme/index.d.ts).

File: index.d.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ declare namespace activeWindow {
 		owner: MacOSOwner;
 
 		/**
-		URL of the active browser tab if the active window is Safari, Chrome, Edge, or Brave.
+		URL of the active browser tab if the active window is Safari, Chrome (includes Beta, Dev, and Canary), Edge (includes Beta, Dev, and Canary), Brave (includes Beta and Nightly), or Opera.
```
Comment (sindresorhus) [inline index.d.ts:77]: Missing `Sidekick`.

And needs to be updated in the readme too.

**Comment by wrgoto** [comment]
Rebased with `main` and added support for beta, dev, and canary. Full list of added browsers

- Brave Browser (Beta and Nightly)
- Google Chrome (Beta, Dev, and Canary)
- Microsoft Edge (Beta, Dev, and Canary)
- Sidekick
- Opera

Brave claims they have a development build, but their site redirects to the beta build.

**Comment by wrgoto** [comment]
Additionally, Opera uses the same app name string for its beta, dev, and canary builds. So you if you have any of them it'll work the same.

**Pull request merged**